CONTRIBUTING.md

```diff
@@ -0,0 +1,20 @@
+# How to contribute
+
+This is an Open edX repo, and we welcome your contributions!
+Please read the [contributing guidelines](http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/process/index.html).
+
+Here are some areas of heightened importance to consider when contributing to the UI Toolkit:
+
+- i18n
+  - Are all user-facing strings tagged?
+  - Is translation work needed?
+- RTL
+  - Will the feature work for right-to-left languages?
+- a11y
+  - Is all functionality usable by using only the keyboard?
+  - Are there any special cases around focus management that need to be addressed?
+- Browser support
+  - Does the feature work at all screen sizes and pixel densities?
+  - Are the browser features this code relies on available in all edX supported browsers?
+- Pattern Library
+  - Does the new feature belong in the Pattern Library instead?
```

Comment:
What does this mean? If the feature has any code then it belongs here. Maybe this should be something more like:

    - If this feature introduces a new pattern then has it been contributed to the Pattern Library repo?